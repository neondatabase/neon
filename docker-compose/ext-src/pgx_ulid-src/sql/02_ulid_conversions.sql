-- Test conversion from timestamptz
SELECT 'timestamptz to ulid conversion' as test_name,
       '2023-03-10 04:00:49.111'::timestamptz::ulid::text = '01GV5PA9EQ0000000000000000' as result;
