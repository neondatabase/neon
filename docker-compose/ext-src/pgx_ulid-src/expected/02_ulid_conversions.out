-- Test conversion from timestamptz
SELECT 'timestamptz to ulid conversion' as test_name,
       '2023-03-10 04:00:49.111'::timestamptz::ulid::text = '01GV5PA9EQ0000000000000000' as result;
           test_name            | result 
--------------------------------+--------
 timestamptz to ulid conversion | t
(1 row)

